samples/drumthumper/src/main/java/com/google/oboe/sample/drumthumper/DrumThumperActivity.kt

```diff
@@ -99,6 +103,26 @@ class DrumThumperActivity : AppCompatActivity(), TriggerPad.DrumPadTriggerListen
         }
     }
 
+    // UI Helpers
+    fun gainPosToGainVal(pos: Int) : Float {
+        // map 0 -> 200 to 0.0f -> 2.0f
+        return pos.toFloat() / 100.0f
```

Comment:
Magic numbers should be moved to constants with descriptive names

Reply:
Done.